tests/test_entityList.py

```diff
@@ -0,0 +1,36 @@
+from os import getcwd
+
+import pytest
+
+from smartsim import Experiment
+from smartsim.entity import Ensemble, EntityList, Model
+from smartsim.settings import RunSettings
+
+
+def test_entityList_init():
+    with pytest.raises(NotImplementedError):
+        ent_list = EntityList("list", getcwd(), perm_strat="all_perm")
+
+
+def test_entityList_type():
+    exp = Experiment("name")
+    ens_settings = RunSettings("python")
+    ensemble = exp.create_ensemble("name", replicas=4, run_settings=ens_settings)
```

Comment:
Since `Ensemble` does not override `EntityList`'s type attribute, this is testing the `EntityList`'s type attribute. Since `EntityList` is an abstract class, this is the most direct way to go about testing the type attribute.

Reply:
This makes sense. Lets place this test in the ``test_ensemble.py`` file however.